.travis.yml

```diff
@@ -25,7 +26,13 @@ install:
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list    ; fi
   - yarn install
   - yarn global add jsonsyn
+  - yarn global add grunt-cli
   - chmod +x script/citest.sh
   
 script:
   - ./script/citest.sh
+
+before_install:
+  - cp .env-travis .env
+  - mysql -e "CREATE TABLE `test`.`trash_bins`( `trashBinID` SMALLINT(5) UNSIGNED NOT NULL AUTO_INCREMENT, `latitude` FLOAT(10,6) UNSIGNED NOT NULL, `longitude` FLOAT(10,6) UNSIGNED NOT NULL, `data` LONGTEXT NOT NULL COLLATE 'utf8mb4_bin', PRIMARY KEY (`trashBinID`), UNIQUE INDEX `latitude_longitude` (`latitude`, `longitude`)) COLLATE='utf8_general_ci' ENGINE=InnoDB ;"
+  - mysql -e "CREATE TABLE `test`.`trash`( `trashID` SMALLINT(5) UNSIGNED NOT NULL AUTO_INCREMENT, `imagePath` VARCHAR(65) NOT NULL DEFAULT '0', `latitude` FLOAT(10,6) UNSIGNED NOT NULL DEFAULT '0', `longitude` FLOAT(10,6) UNSIGNED NOT NULL DEFAULT '0', `comment` TINYTEXT NOT NULL, `flag` MEDIUMINT(8) UNSIGNED NOT NULL DEFAULT '0', `username` VARCHAR(50) NOT NULL, `isDisabled` TINYINT(1) UNSIGNED NULL DEFAULT NULL, PRIMARY KEY (`trashID`)) COLLATE='utf8_general_ci' ENGINE=InnoDB ;"
```

Comment:
You can directly input the `database.sql` file, can't you?

Reply:
No, because you have a little bug in your database file. Should I open a PR with a fix? I will do it. 👍